src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureSupplier {
```

Comment:
The interface name 'FigureSupplier' is appropriate as it represents a supplier of figures, which is a noun. No changes needed here.